package.json

```diff
@@ -94,6 +94,7 @@
     "focus-trap-react": "^10.2.3",
     "form-data": "^4.0.0",
     "fuse.js": "^7.0.0",
+    "html-to-ast": "^0.0.6",
```

Comment:
dit is een typed versie van de npm module die in de description van de ticket staat